docs/grammar/SolidityParser.g4

```diff
@@ -379,7 +384,7 @@ expression:
 	expression LBrack index=expression? RBrack # IndexAccess
 	| expression LBrack startIndex=expression? Colon endIndex=expression? RBrack # IndexRangeAccess
 	| expression Period (identifier | Address) # MemberAccess
-	| expression LBrace (namedArgument (Comma namedArgument)*)? RBrace # FunctionCallOptions
+	| expression LBrace (namedArgument (Comma namedArgument)*) RBrace # FunctionCallOptions
```

Comment:
Unrelated to the PR, but it was inconsistent with the compiler parser.
I added `test/libsolidity/syntaxTests/functionCalls/empty_call_options.sol` which would be accepted by the grammar before this change.